adal/tests/Test.ADAL.NET.UIAutomation/CoreMobileMSALTests.cs

```diff
@@ -0,0 +1,152 @@
+﻿//This is a temp location for this file until the visual studio bug that prevents these tests from working in the MSAL project is resolved.
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Test.Microsoft.Identity.Core.UIAutomation.infrastructure;
+
+namespace Test.MSAL.NET.UIAutomation
+{
+    /// <summary>
+    /// Contains the core test functionality that will be used by Android and iOS tests
+    /// </summary>
+    public static class CoreMobileMSALTests
+    {
+
+        private const string MSGraph = "https://graph.microsoft.com";
+        private const string UIAutomationAppV2 = "1e245a30-49aa-43eb-b9c1-c11b072cc92b";
+
+        /// <summary>
+        /// Runs through the standard acquire token flow
+        /// </summary>
+        /// <param name="controller">The test framework that will execute the test interaction</param>
+        public static void AcquireTokenTest(ITestController controller)
+        {
+            //Get User from Lab
+            var user = controller.GetUser(
```

Comment:
seems that getting a user and clearing cache are repeated many times, can be consolidated

Reply:
#Resolved